dot/network/service.go

```diff
@@ -290,6 +320,47 @@ func (s *Service) collectNetworkMetrics() {
 	}
 }
 
+func (s *Service) getTotalStreams(inbound bool) int64 {
+	var count int64
+	for _, conn := range s.host.h.Network().Conns() {
+		for _, stream := range conn.GetStreams() {
+			if isInbound(stream) && inbound || !isInbound(stream) && !inbound {
```

Comment:
I'm not sure of how the priority works here, but maybe add parentheses just to be safe? :thinking: 

```suggestion
			if (isInbound(stream) && inbound) || (!isInbound(stream) && !inbound) {
```